examples/c/src/cpp/phi3.cpp

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could we share the common code through a header file between c and c++ (and different examples)?